data/json/items/migration.json

```diff
@@ -988,5 +988,215 @@
     "id": "40mm_casing",
     "type": "MIGRATION",
     "replace": "40x46mm_m118_casing"
+  },
+  {
+    "id": [ "360_200_mag", "360_400_mag" ],
+    "type": "MIGRATION",
+    "replace": "a180mag"
+  },
+  {
+    "id": "rm360_carbine",
+    "type": "MIGRATION",
+    "replace": "american_180"
+  },
+  {
+    "id": "lead_stabilizer",
+    "type": "MIGRATION",
+    "replace": "stabilizer"
+  },
+  {
+    "id": "lead_barrel_small",
+    "type": "MIGRATION",
+    "replace": "barrel_small"
+  },
+  {
+    "id": "lead_red_dot_sight",
+    "type": "MIGRATION",
+    "replace": "red_dot_sight"
+  },
+  {
+    "id": "lead_rifle_scope",
+    "type": "MIGRATION",
+    "replace": "rifle_scope"
+  },
+  {
+    "id": "lead_acog_scope",
+    "type": "MIGRATION",
+    "replace": "acog_scope"
+  },
+  {
+    "id": "lead_holo_sight",
+    "type": "MIGRATION",
+    "replace": "holo_sight"
+  },
+  {
+    "id": "lead_laser_sight",
+    "type": "MIGRATION",
+    "replace": "rail_laser_sight"
+  },
+  {
+    "id": "lead_grip",
+    "type": "MIGRATION",
+    "replace": "offset_sight_rail"
+  },
+  {
+    "id": "lead_bipod",
+    "type": "MIGRATION",
+    "replace": "bipod"
+  },
+  {
+    "id": "lead_u_shotgun",
+    "type": "MIGRATION",
+    "replace": "masterkey"
+  },
+  {
+    "id": "lead_glm",
+    "type": "MIGRATION",
+    "replace": "m320_mod"
+  },
+  {
+    "id": "lead_stabilizer",
```

Comment:
The leadworks items appear twice in this file (this one is migrated again on line 1003).